internal/scheduler/adapters/adapters_test.go

```diff
@@ -0,0 +1,162 @@
+package adapters
```

Comment:
This looks much better.  

Looking at CodeCov you are missing a few cases:

```
if priorityByPriorityClassName != nil && !ok {
```
	if podSpec.PreemptionPolicy != nil {
```
These are the code blocks you are missing with your tests.  



if podSpec.PreemptionPolicy != nil {!
36 | preemptionPolicy = string(*podSpec.PreemptionPolicy)
37 | }
38 |  

<br class="Apple-interchange-newline">



if podSpec.PreemptionPolicy != nil {!
36 | preemptionPolicy = string(*podSpec.PreemptionPolicy)
37 | }
38 |  

<br class="Apple-interchange-newline">